ee/clickhouse/models/person.py

```diff
@@ -113,6 +113,36 @@ def delete_person(
     sync_execute(DELETE_PERSON_BY_ID, data)
 
 
+# counts duplicate distinct IDs in clickhouse for a team broken down by day
+def count_duplicate_distinct_ids_for_team(team_id: Union[str, int]) -> Dict:
```

Comment:
Maybe pass in period start and end, so it's similar to other stuff here. I'm not sure the day breakdown is needed. If we have the start/end it makes it easier to change the reporting period in the future for everything together. Looks like we currently run it over all time & I propose we just send it for the reporting period.

Reply:
Though it would be useful to know how many duplicates they have from the past (i.e. before we started reporting this to us), so maybe before: X now: Y split could be good

Reply:
Makes sense, I copied the Cloud query where day breakdown made a lot of sense because I can match data to specific commits. But for self-hosted you are correct that this shouldn't matter much. Maybe total will do.

Reply:
Now we only have the total, could we also have from last period (imagine the total is big, but new duplicates aren't being created <- this tells us we need to do a sync, but otherwise we're good ; vs the new number increasing is more troubling). I know we can just compare the total to the previous, but having the new ones separately makes it easier for analytics, e.g. then we can create some dashboards for ourselves more easily (filter = release >= X & show the new duplicate distinct IDs)

Reply:
done, thanks for pushing